cmd/adminapi/main.go

```diff
@@ -74,6 +75,10 @@ func realMain(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("unable to create ObservabilityExporter provider: %w", err)
 	}
+
```

Comment:
I'm not a fan of putting this everywhere - could we do this in the main observability package?

Reply:
Done, moved inside cache service